bot/information/information.py

```diff
@@ -0,0 +1,43 @@
+def information(
+    chemistry,
+    model,
+    solver,
+    isExperiment,
+    cycle,
+    number,
+    isComparison
+):
+    """
+    Generates tweet text.
+    Parameters:
+        chemistry: dict
+        model: pybamm.BaseModel
+        solver: pybamm.BaseSolver
+        isExperiment: bool
+        cycle: list
+        number: numerical
+        isComparison: bool
+    Returns
+        tweet_text: str
+    """
+
+    if isExperiment:
+        tweet_text = (
+            str(cycle)
+            + " * "
+            + str(number)
+            + " "
+            + str(model)
```

Comment:
use `model.name` or open a PR in pybamm editing the `__str__` method of `BaseModel` to return `model.name`